_posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown

```diff
@@ -0,0 +1,250 @@
+
+```
+#Get cluster Health
+GET _cluster/health
+
+#Get nodes in a cluster
+GET _cat/nodes
+
+#Get node health
+GET _cat/health?v=true
+```
+
+
+As with all software, upgrade is a critical path for the community and customers. If you need help as always open an issue and label them `backward-compatibility`, `1.0.0`:
+
+1. For OpenSearch: [OpenSearch Issues](https://github.com/opensearch-project/OpenSearch/issues).
+2. For a plugin: Use the individual plugin repository.
+3. For all the plugins: [OpenSearch Plugins issues](https://github.com/opensearch-project/opensearch-plugins/issues).
+
+
+In any case you’d like to rollback your upgrade for any reason, the simplest path is to spin up an Open Distro cluster and restore using the snapshot you took earlier. You could follow the same process listed in the snapshot upgrade path above. 
+
+### Recommendation
+
+Now that you read through most of the good stuff, we would recommend these in terms of ease of upgrade:
+
+1. Snapshot Upgrade: This is the safest approach if you can spin up multiple clusters and move your customers to the new endpoint after the upgrade. we would recommend this for beginners. 
+2. Restart Upgrade: This is a moderate approach, as this would take down the whole cluster and you will only know after the whole cluster upgrade to verify everything is happy. we would recommend this for intermediate users or for people who know what they are doing.
+3. Rolling Upgrade: This is a complex approach, as this would take down one node at a time and upgrade them. The good thing about this approach is that you get cluster health feedback while going through the process. we would  recommend this for advanced users.
+
+### Going forward
+
+OpenSearch can be upgraded using a Rolling upgrade process for most minor and major versions so upgrading going forward will not usually interrupt service. Like Kibana OSS, OpenSearch Dashboards does not support rolling upgrades, but it supports restart upgrades. You are able to stop all Open Distro instances, deploy a new OpenSearch Dashboards instance and direct traffic to it.
+
+For more information on upgrading please see our [Upgrading FAQs](https://opensearch.org/faq/#c3).and if you have questions about backward compatibility you can check dB’s post on [Backwards Compatibility FAQ](https://opensearch.org/blog/technical-posts/2021/06/opensearch-backwards-compatibility-faq/).. If you’re looking for a great “getting started” guide I’d recommend Gedalyah’s [Install and Configure OpenSearch guide](https://logz.io/blog/opensearch-tutorial-installation-configuration/).
+
+Closing on a high note, hope this blog post helps and feel free to reach out if you have more questions. Thanks to OpenSearch team for making this happen.
+
+If you have any feedback/suggestions on new topics please reach out to us. we plan to have a follow up post to upgrade from Elasticsearch to OpenSearch for community who do not use Open Distro. 
```

Comment:
Couple of notes;

- Needs a space between "[Upgrading FAQs](https://opensearch.org/faq/#c3)" and ".and if" + we either need to remove the "." or capitalize the "a". 
- Double period ("..") after backwards compatibility FAQ. 
- I'd consolidate the last two sentences. Something like "Hope this blog post helps! We plan to have a follow up post on upgrading from Elasticsearch to OpenSearch next! If you have any feedback/suggestions on new topics please reach out.

Reply:
I like that. Sure just modified a little bit.